daphne/src/vdaf/mod.rs

```diff
@@ -437,53 +477,57 @@ impl VdafConfig {
                 return Err(DapAbort::UnrecognizedMessage);
             }
```

Comment:
Reviewer note: seemingly large change below is actually quite small. Code has been moved around so that it can be shared by both Prio2 and Prio3. Your best bet is to look the file on the base branch and the PR branch side by side.